config/01-ce-auth.conf

```diff
@@ -7,9 +7,14 @@
 #
 ###############################################################################
 
-# Uncomment the following line if your host certificate was issued by an
+# Uncomment the following lines if your host certificate was issued by an
 # IGTF-accredited CA, such as the InCommon IGTF CA or DigiCert Grid CA-1:
 # https://dl.igtf.net/distribution/igtf/current/accredited/accredited.in
 # If you are using a Let's Encrypt certificate, leave this line commented.
 #
-# CE.USE_GRID_CERTS = True
+# AUTH_SSL_SERVER_CERTFILE = /etc/grid-security/hostcert.pem
+# AUTH_SSL_SERVER_KEYFILE = /etc/grid-security/hostkey.pem
+# AUTH_SSL_SERVER_CADIR = /etc/grid-security/certificates
+# AUTH_SSL_CLIENT_CADIR = /etc/grid-security/certificates
```

Comment:
Should the client CA dir really depend on what kind of certificate the server has?

Reply:
Yes because the clients in this case are not clients to the server but rather when the server is acting as a client